lib/db/init.ts

```diff
@@ -0,0 +1,41 @@
+import {db} from "../../lib/db";
+import * as fs from "fs";
+
+export function initDB() {
+    const ELW_COUNT: number = 309;
+    const ECS_COUNT: number = 529;
+    const ELW_ID = 1;
```

Comment:
Why not export these so you can use them in your tests and not have to define it in multiple places?